x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curations_table.tsx

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
The contents of this file are unmodified from what used to be in `curations.tsx` except for imports